src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,41 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person as PersonType } from '../types';
+import { Person } from './Person';
+import { SearchLink } from './SearchLink';
+
+interface Props {
+  people: PersonType[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const sortParams = (value: string) => {
+    if (
+      searchParams.has('sort') &&
+      searchParams.has('order') &&
+      value === sort
+    ) {
+      return { sort: null, order: null };
+    }
+
+    return sort === value
+      ? { sort, order: 'desc' }
+      : { sort: value, order: null };
```

Comment:
The logic in `sortParams` seems incorrect. When the current sort value matches the column, it should toggle between ascending and descending order. The current logic sets the order to 'desc' when the sort matches, which might not toggle correctly. Consider revising this logic to handle toggling between 'asc' and 'desc' properly.